libs/designsystem/page/src/page-footer/page-footer.component.scss

```
@@ -11,7 +12,7 @@ $padding-vertical: utils.size('xxs');
 
   .wrapper {
     position: relative;
-    max-width: var(--page-content-max-width, utils.$page-content-max-width);
+    max-width: var(--page-content-max-width, #{map.get(utils.$page-content-max-widths, 'default')});
```

Consider using variable for readability and simpler usage:
```suggestion
    max-width: var(--page-content-max-width, $page-content-max-width-default);
```